docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -40,7 +46,7 @@ assets = kili.assets("<your_project_id>", fields=["externalId", "latestLabel.jso
 print(assets[0])
 {'latestLabel': {'jsonResponse': {'CLASSIFICATION_JOB': {'categories': [{'name': 'VEHICLE'}]}}}, 'externalId': '0'}
 ```
-  3. You can now get your label this way, and for example write the category name into a `0.txt` file:
+  3. You can now get your label, and write the category name into a `0.txt` file, for example::
 ```python
 for asset in assets:
     if asset["latestLabel"]: # covers the assets without label
```

Comment:
I think we need to explain here that the "0.txt" is set from the externalId.

"and write the category name into a text file named after the externalId "0.txt""